docs/usage/workload_storage.rst

```diff
@@ -0,0 +1,104 @@
+Workload Storage
+================
+
+Some workloads need different volumes with different storage capacities
+to fit its components needs. These volumes are stored in Logical Volumes
+Manager Logical Volumes (:term:`LVM LV`).
+
+
+Volumes
+-------
+
+Following the total storage capacity needed, establish a number of
+volumes and their capacities for each node, as exemplified below:
+
+.. code-block:: yaml
+
+    metalk8s_lvm_drives_vg_metalk8s: ['/dev/vdb']
+    metalk8s_lvm_lvs_vg_metalk8s:
+        lv01:
+            size: 52G
+        lv02:
+            size: 52G
+        lv03:
+            size: 52G
+        lv04:
+            size: 11G
+        lv05:
+            size: 11G
+        lv06:
+            size: 11G
+        lv07:
+            size: 5G
+        lv08:
+            size: 5G
+
+Resize LVs
+----------
+
+Volumes can be resized (one or several at once). Change the volume
+size value to a higher one and run:
+
+.. code::
+
+  ansible-playbook -i <inventory>/hosts -t storage playbooks/deploy.yml
+
+Configuration layout
+--------------------
+
+The configuration can be applied to groups and hosts in two
+different ways.
+
+.. note::
+   Configuration files are merged for every created host or group.
+
+To apply a configuration, create a .yaml file in either (or both)
+:file:`group_vars` and :file:`host_vars` with the group name
+associated, or create a folder in :file:`group_vars` or
+:file:`host_vars` with several .yaml files.
```

Comment:
I don't like the use of `.yaml files`, would prefer `YAML files` (various occurrences).